src/Platform/OpenGL/Shaders/lens_flare_prefilter.glsl

```diff
@@ -0,0 +1,17 @@
+layout(binding = 0) uniform sampler2D inputColor;
+
+uniform float uScale;
+uniform float uBias;
+
+in vec2 TexCoord;
+
+out vec4 outputColor;
+
+void main()
+{
+   vec4 col = texture(inputColor, TexCoord);
+   if(length(max(vec4(0.0),col+vec4(uBias)))<=0.001f)
```

Comment:
do not compute colors like that. You can use average white point (averafe_white_fragment.glsl) to get average luminance. And then check how different is the current point from average

Reply:
That's a great idea!

Reply:
hm.. I changed way to compute color. 🤣But I got impediment using average_white_fragment. Didn't figured out why this represent average lum. And I also can not understand why we only access position <0,0>from it. 😢
https://github.com/asc-community/MxEngine/blob/d534650dddef88467ae0793fe7c35267250c3064/src/Platform/OpenGL/Shaders/hdr_to_ldr_fragment.glsl#L36

Reply:
cause averageWhiteTex size is 1x1

Reply:
but there is no proper downscaling for it

Reply:
I suggested you to use averageWhiteTex directly in you shader

Reply:
Okay. Seems working. Nice suggestion.